Build.cmake

```diff
@@ -120,13 +133,6 @@ ENDIF()
       set_target_properties(${G3LOG_LIBRARY} PROPERTIES MACOSX_RPATH TRUE)
    ENDIF()
 
-   IF(NOT ${ADD_BUILD_WIN_SHARED}.x STREQUAL ".x")
-      IF(NOT(CMAKE_VERSION LESS 3.4) AND MSVC)
```

Comment:
With my CMake it does work. I added test messages inside the if statement and it seemed to do what it was supposed to.